mediapipe/examples/python/ovms_holistic_tracking.py

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import cv2
+import mediapipe as mp
+import time
+import numpy as np
+import threading
+
+class RequestingThread(threading.Thread):
```

Comment:
This is basically repeated script for face detection - why we can't reuse code here?

Reply:
I will remove multithreading so it will be only in one example.